index.bs

```diff
@@ -4776,6 +4782,49 @@ RogueList when performing ECDAA-Verify (see section 3.6 in [[!FIDOEcdaaAlgorithm
 [[FIDOMetadataService]] provides one way to access such information.
 
 
+## Security Benefits for [RPS] ## {#sctn-rp-benefits}
+
+The main benefits offered to [=[RPS]=] by this specification include:
+
+1. Users and accounts can be secured using widely compatible, easy-to-use multi-factor authentication.
+1. The [=[RP]=] does not need to provision [=authenticator=] hardware to its users. Instead, each user can independently obtain
+    any conforming [=authenticator=] and use that same [=authenticator=] with any number of [=[RPS]=]. The [=[RP]=] can optionally
+    enforce requirements on [=authenticators=]' security properties by inspecting the [=attestation statements=] returned from the
+    [=authenticators=].
+1. [=Registration=] and [=authentication=] [=ceremonies=] are highly resistant to [=man-in-the-middle attacks=].
```

Comment:
I cant help feeling uneasy about throwing down the "highly resistant" gauntlet...  I'd be inclined to just say "resistant", or maybe "strongly resistant", or "more resistant than username/password"...

Reply:
Ok, let's go with just "resistant" then. :)